test/blocks/global-footer/global-footer.test.js

```diff
@@ -0,0 +1,266 @@
+
+  describe('basic sanity tests', () => {
+    it('should have footer', async () => {
+      const footer = await createFullGlobalFooter({ waitForDecoration: true });
+      expect(footer).to.exist;
+      expect(document.querySelector('footer')).to.exist;
+    });
+
+    it('should have menu', async () => {
+      const footer = await createFullGlobalFooter({ waitForDecoration: true });
+
+      expect(footer).to.exist;
+
+      const menuSelectorsKeys = Object.keys(menuSelectors);
+      for (const selectorKey of menuSelectorsKeys) {
+        const targetEl = document.querySelector(menuSelectors[selectorKey]);
+        expect(!!targetEl).to.equal(true);
+      }
+    });
+
+    it('should have featured products links', async () => {
+      const footer = await createFullGlobalFooter({ waitForDecoration: true });
+
+      expect(footer).to.exist;
+
+      const featuredProductsSelectorsKeys = Object.keys(featuredProductsSelectors);
+      for (const selectorKey of featuredProductsSelectorsKeys) {
+        const targetEl = document.querySelector(featuredProductsSelectors[selectorKey]);
+        expect(!!targetEl).to.equal(true);
+      }
+    });
+
+    it('should have social links', async () => {
+      const footer = await createFullGlobalFooter({ waitForDecoration: true });
+
+      expect(footer).to.exist;
+
+      const socialSelectorsKeys = Object.keys(socialLinksSelectors);
+      for (const selectorKey of socialSelectorsKeys) {
+        const targetEl = document.querySelector(socialLinksSelectors[selectorKey]);
+        expect(!!targetEl).to.equal(true);
+      }
+    });
+
+    it('should have region picker', async () => {
+      const footer = await createFullGlobalFooter({ waitForDecoration: true });
+
+      expect(footer).to.exist;
+
+      const regionPickerSelectorsKeys = Object.keys(regionPickerSelectors);
+      for (const selectorKey of regionPickerSelectorsKeys) {
+        const targetEl = document.querySelector(regionPickerSelectors[selectorKey]);
+        expect(!!targetEl).to.equal(true);
+      }
+    });
+
+    it('should have legal section', async () => {
+      const footer = await createFullGlobalFooter({ waitForDecoration: true });
+
+      expect(footer).to.exist;
+
+      const legalSectionSelectorsKeys = Object.keys(legalSelectors);
+      for (const selectorKey of legalSectionSelectorsKeys) {
+        const targetEl = document.querySelector(legalSelectors[selectorKey]);
+        expect(!!targetEl).to.equal(true);
+      }
+    });
+
+    it('should render the whole footer', async () => {
+      const footer = await createFullGlobalFooter({ waitForDecoration: true });
+      expect(footer).to.exist;
+
+      for (const selectorKey of allSelectorsKeys) {
+        const targetEl = document.querySelector(selectors[selectorKey]);
+        expect(!!targetEl).to.equal(true);
+      }
+    });
+  });
+
+  describe('conditional render tests', () => {
+    it('should render the footer when in viewport', async () => {
```

Comment:
This is the same test as your first ones, correct? If yes, we can skip on this first one since we already tested the functionality